### ISSUE TITLE
Vercel does not render correctly when adding a file.

### ISSUE THREAD

**Issue by raincandy** [shuding/nextra#400]
Hi @shuding,

When I try to add an post to a directory and update the corresponding `meta.json`, the left list of other posts in that directory does not update. But I can access this new post directly from the URL, and the left-hand list of the new post is rendered correctly. This issue can be resolved by going into Vercel and manually redeploying and unchecking `Redeploy with existing Build Cache.`, but it doesn't make sense. Is there a good solution for me?

Sorry for not using the latest version, as the i18n API seems to have changed in the latest version and updating directly to the latest version will report an error like this:

```
Module not found: Can't resolve 'nextra/locales'
> 1 | import { locales } from "nextra/locales"
  2 | export const middleware = locales
  3 |
```

At the same time, the latest version does not support placing articles in the `src/pages` directory. https://github.com/shuding/nextra/issues/356

This is the version that Vercel builds automatically: https://onedrive-vercel-index-docs-l5v6w1v6n-shiroha.vercel.app/
And this is the manually redeployed version: https://onedrive-vercel-index-docs-o9pqmekcn-shiroha.vercel.app/
The new post's URL is `/blog/first-blog`, and there is another article in this directory: `/blog/whats-new`.

In the automatic build version, left list of `whats-new` page is:
![auto-whats-new](https://user-images.githubusercontent.com/59050289/157839075-f185045d-8bd9-4928-a3d2-bf607866a0e5.png)
But when I access new post by URL, left list of the new post is:
![auto-new-post](https://user-images.githubusercontent.com/59050289/157839316-33228c93-1021-4cec-85d8-eabd39ce1cfc.png)
After manually redeploy, left list of `whats-new` is normal:
![after-redeploy](https://user-images.githubusercontent.com/59050289/157839457-7c0ee679-3e2e-4922-bb87-a92f650fe969.png)

Package manager: `pnpm 6.32.3`

Node Version: v14.17.6

Dependencies:

```json
"dependencies": {
    "crypto-js": "^4.1.1",
    "next": "^12.0.10",
    "nextra": "^2.0.0-alpha.23",
    "nextra-theme-docs": "^2.0.0-alpha.23",
    "react": "^17.0.2",
    "react-dom": "^17.0.2",
    "react-intersection-observer": "^8.33.1"
  },
  "devDependencies": {
    "@types/react": "^17.0.39",
    "eslint": "^8.9.0",
    "eslint-config-next": "^12.1.0",
    "typescript": "^4.5.5"
  }
```

Repo: https://github.com/raincandy/onedrive-vercel-index-docs

**Comment by shuding** [comment]
Thanks for opening this! This bug should have been fixed with #370, try upgrade `nextra` and `nextra-theme-docs` to the latest alpha (there are a few breaking changes but I opened https://github.com/spencerwooo/onedrive-vercel-index-docs/issues/7, let me know!).

**Comment by raincandy** [comment]
Thanks for reply!
